shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -2238,31 +2238,31 @@ public boolean isFailure() {
         final String segment;
         @ToString.Exclude
         final String delegationToken;
-        final long fromVersion;
+        final long fromPosition;
         final int suggestedEntryCount;
 
         @Override
         public void process(RequestProcessor cp) {
-
+            cp.readTableEntriesDelta(this);
         }
 
         @Override
         public void writeFields(DataOutput out) throws IOException {
             out.writeLong(requestId);
             out.writeUTF(segment);
             out.writeUTF(delegationToken == null ? "" : delegationToken);
-            out.writeLong(fromVersion);
+            out.writeLong(fromPosition);
             out.writeInt(suggestedEntryCount);
         }
 
         public static WireCommand readFrom(ByteBufInputStream in, int length) throws IOException {
             long requestId = in.readLong();
             String segment = in.readUTF();
             String delegationToken = in.readUTF();
-            long fromVersion = in.readLong();
+            long fromPosition = in.readLong();
             int suggestedEntryCount = in.readInt();
 
-            return new ReadTableEntriesDelta(requestId, segment, delegationToken, fromVersion, suggestedEntryCount);
+            return new ReadTableEntriesDelta(requestId, segment, delegationToken, fromPosition, suggestedEntryCount);
```

Comment:
you need to add `.requireRelease()` before returning this object. See the changes that were made to `TableEntriesRead` ,for example.

Reply:
Which resources need releasing? I extended `TableEntriesReadDelta` to release its `entries` but it didn't seem like `ReadTableEntriesDelta` required it.

Reply:
My bad. This comment relates to line 2303. See below.